templates/repo/editor/edit.tmpl

```diff
@@ -38,9 +38,9 @@
 						data-url="{{.Repository.Link}}/markup"
 						data-context="{{.RepoLink}}"
 						data-previewable-extensions="{{.PreviewableExtensions}}"
-						data-line-wrap-extensions="{{.LineWrapExtensions}}">
-{{.FileContent}}</textarea>
+						data-line-wrap-extensions="{{.LineWrapExtensions}}"></textarea>
 					<div class="editor-loading is-loading"></div>
+					<script>window.monacoContent = "{{.FileContent}}";</script>
```

Comment:
I would prefer to use `pageData` for this case, to avoid adding more variables to `window` object.